website/content/docs/nia/configuration.mdx

```diff
@@ -166,12 +166,12 @@ task {
   providers = ["my-provider"]
   source = "path/to/services-condition-module"
 
-  // optional: declare empty services condition block
   condition "services" {
-    // no additional configuration options currently
+    regexp = "^web.*"
```

Comment:
The below detail on `regexp` states that "if regexp is configured, then task.services must be omitted or empty. But this example now has both `services` and `regexp` set.

Can you update this example code block to be one or the other but not both?

Reply:
Another place where dedicated example blocks can be on the other page `/docs/nia/tasks#services-condition` similar to how Catalog-Services condition has an example block.

Reply:
Oh whoops, will make the changes!

Reply:
Also added an example block showing that the empty service condition block is the same as no condition block.

Reply:
It's clear! Thanks for the update